keyboards/xdboards/recon/keyboard.json

```
@@ -0,0 +1,206 @@
+{
+    "manufacturer": "XD Boards",
+    "keyboard_name": "xdboards/recon",
+    "maintainer": "Xanimos",
+    "development_board": "promicro",
+    "pin_compatible": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "oled": true
+    },
+    "matrix_pins": {
+        "cols": ["D4", "C6", "D7", "E6", "B4", "B5"],
+        "rows": ["F4", "F5", "F6", "F7", "B1"]
+    },
+    "url": "https://xdboards.xyz",
+    "tags": ["columnar", "split", "thumb cluster", "rgb", "encoder"],
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x02F7",
+        "vid": "0xFEED"
+    },
+    "encoder": {
+        "rotary": [
+            { "pin_a": "B6","pin_b": "B2" }
+        ]
+    },
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "D3",
+        "handedness": {
+            "pin": "D2"
+        },
+        "matrix_pins": {
+            "right": {
+                "cols": ["D4", "C6", "D7", "E6", "B4", "B5"],
+                "rows": ["F4", "F5", "F6", "F7", "B1"]
+            }
+        },
+        "transport": {
+            "sync": {
+                "layer_state": true,
+                "oled": true,
+                "modifiers": true
```

You probbably want matrix state, too, if you have rgb matrix enabled.
```suggestion
                "matrix_state": true,
                "modifiers": true
```